src/ToggleButton/Option.js

```diff
@@ -0,0 +1,253 @@
+// @flow
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import ButtonBase from '../internal/ButtonBase';
+import Menu from '../Menu';
+import ArrowDropdown from '../svg-icons/arrow-drop-down';
+import withStyles from '../styles/withStyles';
+import createStyleSheet from '../styles/createStyleSheet';
+import { fade } from '../styles/colorManipulator';
+import common from '../colors/common';
+import grey from '../colors/grey';
+
+/**
+ * Represents an option within a Toggle Button
+ *
+ */
+
+export const styleSheet = createStyleSheet('MuiOption', {
+  root: {
+    backgroundColor: 'transparent',
+    borderRadius: 0,
+    fontWeight: 600,
+    fontSize: 14,
+    float: 'left',
+    padding: '0px 12px',
+    textTransform: 'uppercase',
+    // borderLeft: props.optionStyle.borderLeft? props.optionStyle.borderLeft : 'none',
+    // borderRight: props.optionStyle.borderRight? props.optionStyle.borderRight : 'none',
+    height: 36,
+  },
+  button: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    color: fade(common.black, 0.3),
```

Comment:
I am now using the text hint color, however it looks slightly darker than shown in the image. I hope that this is what is desired.